src/views/Timesheets/index.js

```diff
@@ -138,13 +138,13 @@ const Timesheets = props => {
     if (timeIn !== null && timeOut !== null) {
       checkForFutureDate(timeIn, timeOut);
       let timeDiff = timeOut.getTime() - timeIn.getTime();
-      let calculatedTimeDiff = timeDiff / 1000 / 60 / 60;
-      let roundedHourDifference = 0;
-      if (calculatedTimeDiff > 0 && calculatedTimeDiff < 0.1) {
-        roundedHourDifference = 0.1;
-      } else if (calculatedTimeDiff >= 0.1) {
-        roundedHourDifference = (Math.round(calculatedTimeDiff * 10) / 10).toFixed(2);
+      let calculatedTimeDiff = timeDiff / 3600000; //3,600,000 milliseconds in an hour.
+      let roundedHourDifference = (Math.round(calculatedTimeDiff * 12) / 12).toFixed(2);
```

Comment:
This reorganization of the code is a nice improvement.

However, is the "toFixed(2)" needed?  I don't think so, and the code would be a little clearer without it, so a further small improvement might be possible.

Reply:
I think the .tofixed(2) is necessary to display a result number value of such as 1 as a text value of 1.00 on the screen.  Although I'm not 100% sure.  In terms of rounding and database insertion it's definitely not.

Reply:
Ah, that sounds reasonable.  Thanks.